### PR TITLE
PMS-4 View Item Details (file convention relabel)

### PR DESCRIPTION
**Pull request by erik-lance**
This relabels getItem to getItems since it is a db.findMany function. I needed to relabel this since I created another function simply for getting ONE item which is now the new getItem.

This will also be pushing the feature of double-clicking an item goes to its page.

### CODE REVIEW

File: views/item.hbs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        {{! <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/about">About</a></li>
+            <li><a href="/contact">Contact</a></li>
+        </ul> }}
+        <div id="main-logo">
+            <img src="/img/Titafinds/icon.png" id="icon-logo" />
+        </div>
+    </nav>
+</header>
```
Comment (aar0npal) [inline views/item.hbs:12]: Wait, hindi ba tanggal ang navbar? Or may secret access ka sa mockup 👀

File: controllers/itemController.js [hunk 2]
```diff
@@ -15,10 +15,19 @@ const itemController = {
     },
 
     // Redirects to home page
-    homeRedirect: function (req, res) {
+    homeRedirect: async function (req, res) {
```
Comment (aar0npal) [inline controllers/itemController.js]: Curious lang, why add async?

File: public/js/index.js [hunk 3]
```diff
@@ -149,6 +149,14 @@ $(function () {
             },
         ],
         records: Items,
+        onDblClick: function(recid) {
```
Comment (aar0npal) [inline public/js/index.js:152]: Baka masyado lang siguro akong maarte 🤣, but I think lets make it consistent by making the parameter camelCase. Lagay ko nalang din siguro sa coding standard.

File: public/js/item.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+var page_item = null;
+
+function Item(image, name, code, type, classification, length, size, weight, quantity, sellingType, purchasePrice, sellingPrice, status) {
```
Comment (aar0npal) [inline public/js/item.js:6]: Missing js doc. I think meron na siya sa recent commit (sa index.js), you can copy+paste it nalang.

File: public/js/item.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        weight: weight,
+        quantity: quantity,
+        sellingType: sellingType,
+        purchasePrice: purchasePrice,
+        sellingPrice: sellingPrice,
+        status: status,
+    };
+}
+
+function getItem() {
```
Comment (aar0npal) [inline public/js/item.js:28]: Missing js doc.

File: public/js/item.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            for (var i = 0; i < num_keys; i++) 
+            {
+                $('#fields-table').append(item_desc(fields[i], values[i]))
+            }
+
+        },
+    });
+}
+
+function item_desc(field, desc) {
```
Comment (aar0npal) [inline public/js/item.js]: Missing js doc. And also, I'm not sure pero what do you prefer? I prefer camelCase for function namings para consistent din (like helper functions inside utils.js).

**Review by aar0npal — CHANGES_REQUESTED** [review]

File: views/item.hbs (hunk 1 above)
Comment (erik-lance) [reply views/item.hbs:12]: Kinopya ko lang yung other hbs file HAHA

File: controllers/itemController.js (hunk 2 above)
Comment (erik-lance) [reply controllers/itemController.js]: I don't remember adding this HAHAHAH I could've sworn it was always like that, baka nalagyan ko pwede nating tanggaling

File: public/js/index.js (hunk 3 above)
Comment (erik-lance) [reply public/js/index.js:152]: Wait so tama naman diba HAHA tiyaka akala ko strict ata w2ui sa methods nila

Comment (aar0npal) [reply public/js/index.js:152]: Yung recid to recordId

File: controllers/itemController.js (hunk 2 above)
Comment (aar0npal) [reply controllers/itemController.js]: Remove nlng

File: public/js/item.js (hunk 6 above)
Comment (erik-lance) [reply public/js/item.js]: Refer to 1c5832c0961fe51ecce3eb7cf13a7be74a83ac02

File: public/js/item.js (hunk 5 above)
Comment (erik-lance) [reply public/js/item.js:28]: Refer to 1c5832c0961fe51ecce3eb7cf13a7be74a83ac02

File: public/js/item.js (hunk 4 above)
Comment (erik-lance) [reply public/js/item.js:6]: Refer to 1c5832c0961fe51ecce3eb7cf13a7be74a83ac02

**Review by aar0npal — APPROVED** [review]
Wow POGGERS

**Pull request merged**